### PR TITLE
WIP: These changes should resolve the following issues : #73,#35,#28,#80

### PR DESCRIPTION
**Pull request by ghalym**
These changes should address the following issues : #73,#35,#28,#80

### CODE REVIEW

File: lcls-twincat-pmps/PMPS/MajorComponents/FastFaults/FB_FastFault.TcPOU [hunk 1]
```diff
@@ -16,6 +16,10 @@ FUNCTION_BLOCK FB_FastFault
 VAR_INPUT
 
     i_xOK        :    BOOL; // Connect to fast-fault condition (false produces fault)
+	{attribute 'pytmc' := '
+        pv: Reset
+        io: io
+     '}
```
Comment (ZLLentz) [inline lcls-twincat-pmps/PMPS/MajorComponents/FastFaults/FB_FastFault.TcPOU:22]: Should remove this until we decide what we want to do with pragmas here

**Comment by ZLLentz** [comment]
closes #65

**Comment by ZLLentz** [comment]
The following files also reference a 16-bit eV range and need a look:
- https://github.com/pcdshub/lcls-twincat-pmps/blob/1974e33f5c559bea3e50258213397855def32b4c/lcls-twincat-pmps/PMPS/HelperFunctions/F_eVRangeCalculator.TcPOU#L27
- https://github.com/pcdshub/lcls-twincat-pmps/blob/1974e33f5c559bea3e50258213397855def32b4c/lcls-twincat-pmps/PMPS/HelperFunctions/F_eVWithinSpec.TcPOU#L8
- https://github.com/pcdshub/lcls-twincat-pmps/blob/1974e33f5c559bea3e50258213397855def32b4c/lcls-twincat-pmps/PMPS/HelperFunctions/F_SetBeamParams.TcPOU#L8
- https://github.com/pcdshub/lcls-twincat-pmps/blob/1974e33f5c559bea3e50258213397855def32b4c/lcls-twincat-pmps/PMPS/MajorComponents/FB_BPControlDevice.TcPOU#L33
- https://github.com/pcdshub/lcls-twincat-pmps/blob/1974e33f5c559bea3e50258213397855def32b4c/lcls-twincat-pmps/PMPS/TestComponents/FB_evRangeCalculator_Test.TcPOU#L25

**Comment by ZLLentz** [comment]
I don't see where this addresses #73

**Comment by ZLLentz** [comment]
Reminder that we need to update this page slightly after deploying: https://confluence.slac.stanford.edu/display/L2SI/PMPS+Photon+Energy

**Comment by ZLLentz** [comment]
Are we planning to merge/tag/deploy this before the end of the pamm?

**Comment by ghalym** [comment]
Yes, I was hoping we can start with this process tomrrow.

**Pull request merged**